content/src/main/content/jcr_root/apps/acs-samples/angularjs-integration/clientlibs/footerLib/js.txt

```diff
@@ -0,0 +1,13 @@
+#base=js
+
+angular/core/main.js
+angular/core/baseController.js
+angular/core/routingService.js
+
+angular/partialControllers/homePartialController.js
```

Comment:
Is "partialController" naming an angularJS convention? Are these any different from normal controllers (dont these just inherit from a base controller)

Just seeing if we can reduce the number of points that have the potential to confuse (case in point; I am wondering why this isnt just named "controllers")